runtime/oti/jvminit.h

```diff
@@ -391,6 +391,7 @@ enum INIT_STAGE {
 #define VMOPT_XXDISABLEUTFCACHE "-XX:-UTFCache"
 #define VMOPT_XXENABLEENSUREHASHED "-XX:+EnsureHashed:"
 #define VMOPT_XXDISABLEENSUREHASHED "-XX:-EnsureHashed:"
+#define VMPOT_XXNOOPENJ9COMMANDLINEENV "-XX:NoOpenJ9CommandLineEnv"
```

Comment:
Perhaps this should be `-XX:-OpenJ9CommandLineEnv` for consistency?

Reply:
Agree. And in that case we should support -XX:+OpenJ9CommandLineEnv as well.